xcdat/temporal.py

```diff
@@ -739,6 +738,9 @@ def _set_obj_attrs(
         ValueError
             If an incorrect ``dec_mode`` arg was passed.
         """
+        # The time bounds.
+        self._time_bounds = self._dataset.bounds.get_bounds("T")
+
```

Comment:
I had this line here originally, but made the silly mistake of moving it to the class`__init__` method in another PR which caused this bug.